mono/tests/Makefile.am

```diff
@@ -1170,7 +1236,19 @@ test-env-options:
 
 # Precompile the test assemblies in parallel
 compile-tests:
-	$(MAKE) -j4 $(TESTS_CS) $(TESTS_IL) $(TESTS_BENCH) $(TESTS_STRESS) $(TESTS_GSHARED) $(TESTSAOT_CS) $(TESTSAOT_IL) $(TESTSAOT_BENCH) $(TESTSAOT_STRESS) $(TESTSAOT_GSHARED) compile-gac-loading
+	$(MAKE) -j4 $(TESTS_CS) \
```

Comment:
noise, unrelated.

Reply:
I added to this line. It is not readable without the newlines.

Reply:
guess I can do better w/o diff by adding a dependency on the outputs
I use make -j already in the outer layer and this internal make -j is not a good idea really -- too many schedulers and fan outs.